### PR TITLE
Byebye skbio

### PR DESCRIPTION
**Pull request by amnona**
make skbio dependency optional (only when using the dendrogram tree)
added to test optional dependency since we have a dendrogram unit test

### CODE REVIEW

**Comment by amnona** [comment]
@RNAer for some reason travis is not running the tests. any idea why?
also, can you go over and comment/merge?

File: setup.py [hunk 1]
```diff
@@ -65,11 +65,12 @@
...
           'biom-format',
           'statsmodels',
           'ipywidgets',
           'docrep'],
-      extras_require={'test': ["nose", "pep8", "flake8"],
+      extras_require={'test': ["nose", "pep8", "flake8", 'scikit-bio >= 0.5.1'],
                       'coverage': ["coveralls"],
-                      'doc': ["Sphinx >= 1.4", "sphinx-autodoc-typehints", "nbsphinx"]})
+                      'doc': ["Sphinx >= 1.4", "sphinx-autodoc-typehints", "nbsphinx"],
+                      'dendrogram': ['scikit-bio >= 0.5.1']
```
Comment (RNAer) [inline setup.py:75]: you need to update travis setting to add `dendrogram` installation? to make sure that  the test code for dendrogram and plot_tree is tested

Comment (amnona) [reply setup.py:75]: I think it already contains the skbio requirement (in ci/conda_requirements.txt) so all tests should run ok.
But github is still not running the travis on this PR. any idea why?

Comment (amnona) [reply setup.py:75]: I see it passed on travis:
https://travis-ci.org/github/amnona/calour/builds/746442457
but don't see it here... what's going on?

**Pull request merged**